gpMgmt/bin/gppylib/test/unit/data/gprebalance/unbalanced_to_spread_pos.array

```diff
@@ -0,0 +1,17 @@
+# Coordinator
+1|-1|p|p|s|u|cdw|cdw|5432|/data/coordinator
+# SDW0
+2|0|p|p|s|u|sdw0|sdw0|40000|/data/primary0
+3|1|p|p|s|u|sdw0|sdw0|40001|/data/primary1
+4|2|p|p|s|u|sdw0|sdw0|40002|/data/primary2
+5|3|p|p|s|u|sdw0|sdw0|40003|/data/primary3
+6|4|m|m|s|u|sdw0|sdw0|50000|/data/mirror4
+7|5|m|m|s|u|sdw0|sdw0|50001|/data/mirror5
+# SDW1
+6|4|p|p|s|u|sdw1|sdw1|40000|/data/primary4
+2|0|m|m|s|u|sdw1|sdw1|50000|/data/mirror0
+3|1|m|m|s|u|sdw1|sdw1|50001|/data/mirror1
+# SDW2
+7|5|p|p|s|u|sdw2|sdw2|40000|/data/primary5
+4|2|m|m|s|u|sdw2|sdw2|50000|/data/mirror2
+5|3|m|m|s|u|sdw2|sdw2|50001|/data/mirror3
```

Comment:
Why does the last digit in the ports and names in this file not match, while in the other files it does?